client/src/util/geo_data_util.ts

```diff
@@ -36,4 +52,26 @@ export class GeoDataUtil {
     }
     return options;
   }
+
+  /**
+   * Gives the city which contains the given lat, long if it is a supported city
+   * (Richmond, New Orleans, Toledo).
+   */
+  static getCityForLatLong(latString: string | undefined, longString: string | undefined): City | null {
+    const lat = Number(latString);
```

Comment:
Curious if I should be using this instead of parseInt or float? Not blocking

Reply:
I think they are pretty much the same!